### PR TITLE
Rpmci

### PR DESCRIPTION
**Pull request by pmmccorm**

### CODE REVIEW

**Comment by pmmccorm** [comment]
RFC for now.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-sudo: false
```
Comment (shefty) [inline .travis.yml]: Please keep this false, so we can run in containers.

**Comment by shefty** [comment]
I'm not sure we want to test the RPM build on every PR.  I would rather see some sort of nightly build script do this.

**Comment by pmmccorm** [comment]
The newest version of the config builds RPMs with containers. The RPM takes about 12 seconds to build, why not just check this on each build?

**Comment by shefty** [comment]
I can live with 12 seconds.

File: .travis.yml [hunk 2]
```diff
@@ -6,22 +6,23 @@ compiler:
...
-      compiler: gcc
+addons:
+        apt:
+                packages:
+                        - rpm
 install:
         - ./autogen.sh
         - ./configure --prefix=$HOME
-        - make
+        - make -j2
```
Comment (shefty) [inline .travis.yml]: Why j2?

File: .travis.yml [hunk 3]
```diff
@@ -6,22 +6,23 @@ compiler:
 os:
     - linux
     - osx
-matrix:
-  exclude:
-    - os: osx
-      compiler: gcc
```
Comment (shefty) [inline .travis.yml]: @hppritcha - I think Howard wrote this.  Is there a reason why this was excluded originally?

File: .travis.yml (hunk 2 above)
Comment (pmmccorm) [inline .travis.yml]: http://docs.travis-ci.com/user/ci-environment/

"Cores, up to 2, bursted"

Comment (shefty) [inline .travis.yml]: Ah - got it - thanks

File: .travis.yml (hunk 3 above)
Comment (hppritcha) [inline .travis.yml]: to speed the travis check up some.  I didn't see a need for os-x to double check clang/gcc.

**Comment by shefty** [comment]
Are we good here as well?  I agree that we can revisit the tests if things start taking too long.

**Comment by pmmccorm** [comment]
let me rebase this real quick..

**Comment by pmmccorm** [comment]
Should be good now.

**Pull request merged**